### PR TITLE
aimzOG COURSE-PROJECT

### PR DESCRIPTION
**Pull request by aimzOG**

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Neuton" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Work+Sans" rel="stylesheet">
+  <link rel="stylesheet" href="styles/styles.css">
+  <title>Blog | Adoptadog</title>
+</head>
+
+<body>
+  <header>
+    <div class="checkout">
```
Comment (jimmynono) [inline blog.html]: You'll want to figure out a better way to lay these out so they live next to each other. Some ideas, flex, grid or positioning

File: styles/dogs.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.dogs {
+    display: grid;
```
Comment (jimmynono) [inline styles/dogs.css]: Watch your indentation here, looks like 4 spaces rather than the two you use in the other files

File: styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+body {
+  padding: 0 2rem;
+  font-family: "Work Sans", serif;
+}
+
+header {
+  background-image: url("../images/header-image.jpg");
+  background-size: cover;
+  color: #ffffff;
```
Comment (jimmynono) [inline styles/styles.css]: It's a minor thing, but when using colors keep the same case across the files. My preference is all uppercase, but either is fine, but you use both all uppercase and all lower case in this file

File: styles/styles.css [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+.about, .blog {
+  display: flex;
+  margin: 3rem 0;
+}
+
+.intro, .blog-text {
+  padding-left: 3rem;
+}
+.home-dogs {
```
Comment (jimmynono) [inline styles/styles.css]: Make sure you have extra lines between all of your selectors

File: styles/styles.css (hunk 3 above)
Comment (aimzOG) [reply styles/styles.css]: ooo, that's the copy and paste getting me. thanks!

**Comment by jaamp** [comment]
Hi - here is my peer review - John

Hi,
Overall your site looks very clean and functional.  I like on the home page’s responsiveness. In the CSS good use of em and rem, and by minimal use of px.  Back ground color and coloring in general is non-obtrusive allowing the focus on the photos and the cards in general.  You have error validation that works very well and the form resizing works great.  
I think one of the main points of improvement (and is really only a preference, but I think rounding the edges of the photos would soften views and still give a good clean appearance.
Again, your site is very clean, functional and the responsiveness is perfect.  Really great job.
John

**Pull request closed without merging**